lib/internal/process/coverage.js

```diff
@@ -28,8 +28,8 @@ function writeCoverage() {
 
   try {
     session.post('Profiler.takePreciseCoverage', (err, coverageInfo) => {
+      if (err) console.error(err);
```

Comment:
If an error exists, the code should not continue, otherwise the write would still be triggered.

Adding a return statement fixes this: `if (err) return console.error(err);`